src/silkaj.py

```diff
@@ -31,6 +31,8 @@ def usage():
     \n \
     \n - network: Display current network with many information \
     \n      --discover     Discover all network (could take a while) \
+    \n      -s, --sort     Sort column names comma-separated (for example \"-s block,diffi\") \
```

Comment:
There shouldn't be any space here since it is a "comma-separated" list of items that could be listed on command line such as:
./silkaj network -s block,uid
where
./silkaj network -s block, uid
does NOT work. I should add a check that these don't end with a comma, otherwise it means the user has put a space, which is not good.